frontend/dispatcher.lua

```diff
@@ -174,7 +176,6 @@ local settingsList = {
     --
 
     toggle_inverse_reading_order = {category="none", event="ToggleReadingOrder", title=_("Toggle page turn direction"), reader=true, separator=true},
-    swap_page_turn_buttons = {category="none", event="SwapPageTurnButtons", title=_("Invert page turn buttons"), reader=true, condition=Device:hasKeys(), separator=true},
```

Comment:
You moved "invert page turn buttons" from the section related to reading order to the ones with hardware features - fine with that if it's a user global choice (ie. holding the device in the user left hand and having buttons on the side do their things).
Are there devices where physical buttons are not on the side, but on the bottom - where their ordering may indeed depends on reading order?

Reply:
This is supposed to (as best as possible) match the actual layout of the menus, so since I moved the menu, that one had to follow too.

IIRC, the rM 1 may have had funky button placement, but that's another kettle of fish, so, irrelevant here, as the code doesn't care.